source/_patterns/01-molecules/components/article-section.mustache

```diff
@@ -7,6 +7,9 @@
 
   <header class="article-section__header">
     <h{{headingLevel}} class="article-section__header_text">{{{title}}}</h{{headingLevel}}>
+    {{#headerLink}}
+      <a href="{{{url}}}" class="article-section__header_link">{{name}}</a>
```

Comment:
`url` shouldn't be escaped and `name` should be?

Reply:
That's what this does. Assumes no HTML in `name`.

Reply:
Ok if `name` is mean to be a plain string, but `url` needs to be too.

Reply:
👍  Will escape `url` too.